app/src/pages/api/attachment/upload-url.ts

```diff
@@ -0,0 +1,29 @@
+import S3 from 'aws-sdk/clients/s3'
+import { NextApiRequest, NextApiResponse } from 'next'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  console.log("Using BUCKET_NAME:", process.env.BUCKET_NAME);
+  const s3 = new S3({
+    apiVersion: '2006-03-01',
+    accessKeyId: process.env.CL_AWS_ACCESS_KEY,
+    secretAccessKey: process.env.CL_AWS_SECRET_ACCESS_KEY,
+    region: process.env.AWS_REGION
+  })
```

Comment:
Nice S3 setup! To make it easier to use in other places, can you make an `IS3Client`, `S3ApiClient`, and `S3DbClient` to? You can look at the examples for guestbook and report clients, and read more about the motivation here: https://github.com/CodeDayLabs311/cl311/pull/46